src/Input/Input.js

```diff
@@ -67,6 +67,12 @@ export const styleSheet = createStyleSheet('MuiInput', (theme) => {
         appearance: 'none',
       },
     },
+    multiLine: {
+      resize: 'none',
```

Comment:
I used [v0](http://www.material-ui.com/#/components/text-field) as the spec for this, which doesn't allow  manual resizing for multiline text fields. 

I should've used [the official spec](http://www.material-ui.com/#/components/text-field) though...

Which says multiline text fields should work like this: 

![multilinetext](https://cloud.githubusercontent.com/assets/3895824/24837829/9e1d16dc-1d0a-11e7-8ce5-1ccb7176cd6f.gif)

Note that the spec also talks about Text Areas which are different from multi-line text fields, and which I think we'll also want to implement in a separate PR. 

![screen shot 2017-04-09 at 9 56 52 am](https://cloud.githubusercontent.com/assets/3895824/24837841/e247599e-1d0a-11e7-8377-48dc1852f32a.png)